### DISCUSSION TITLE
Is PyCaret well suited for image classification ?

### DISCUSSION THREAD

**Discussion by Ouassim-boukhennoufa** [pycaret/pycaret#4019] [Q&A]
Dear all,

1- I have a question regarding to PyCaret and image classification, as the type of data supported by that framework is dataframe, we can easily convert image to a dataframe and add the label column...but is it well suited ? as the column 0 for instance represents the pixel value and it's independent of the next image, I mean there is no relation between the pixel 0 in image 1 and image 2...too image N. It is not like any other feature, like for instance level of calcium in blood, where here the column represents the same domain for all the data...

2- My second question is: when I am applying it for my image classification task, it gives very good results with the used data for train and test, but when I use it for real data test, once I export the model, it performs very bad. The problem is in the image normalization, as when I change the methodology the results are highly impacted. Is there anyway to manage this ?

**Comment by celestinoxp** [comment]
How many images do you have in the dataframe?
Can you provide more details like the data code and dataframe?
